clang/lib/Driver/ToolChains/Flang.cpp

```diff
@@ -411,6 +412,13 @@ void Flang::addTargetOptions(const ArgList &Args,
   }
 
   // TODO: Add target specific flags, ABI, mtune option etc.
+  if (const Arg *A = Args.getLastArg(options::OPT_mtune_EQ)) {
+    CmdArgs.push_back("-tune-cpu");
+    if (strcmp(A->getValue(), "native") == 0)
```

Comment:
Nit: `strcmp` is a C function. I assume that `A->getValue()` returns a StringRef, or something else that supports normal `==` comparison.

Reply:
I see your point, however when I build the code using normal `==` comparison I get a warning:
`warning: comparison with string literal results in unspecified behavior`
and I am inclined to avoid such possible unspecified behavior.

Reply:
`A` is an `llvm::Arg`. The `getValue()` method [returns](https://llvm.org/doxygen/classllvm_1_1opt_1_1Arg.html#ac268590692356db84db78050196b4940) a `const char*`, so unless it is wrapped in a `llvm::StringRef`, the `==` operator cannot be used for string comparison.

It is surprising though. I, too, would have expected it to return a `llvm::StringRef`.

Reply:
A few lines further up, the string literal get's wrapped into a `StringRef`, which seems to work. 

`if (A->getValue() == StringRef{"Accelerate"})`

Reply:
Thank you for that!  I have made the requested update and there was no build warning.